source/features/bypass-checks.tsx

```diff
@@ -13,14 +15,22 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 	if (directLink) {
 		detailsLink.href = directLink.href;
 	}
-}
+});
 
 async function init(): Promise<void> {
 	// This selector excludes URLs that are already external
-	const thirdPartyApps = select.all<HTMLAnchorElement>('a:not([href="/apps/github-actions"]) ~ div .status-actions[href^="/"]');
+	const thirdPartyApps = [
+		`a:not([href="/apps/github-actions"]) ~ div .status-actions[href^="${location.origin}"]:not(.rgh-bypass-link)`, // Hovercard status checks
+		'a:not([href="/apps/github-actions"]) ~ div .status-actions[href^="/"]:not(.rgh-bypass-link)'
+	].join();
 
-	// If anything errors, RGH will display the error next to the feature name
-	await Promise.all(thirdPartyApps.map(bypass));
+	observe(thirdPartyApps, {
+		constructor: HTMLAnchorElement,
+		async add(thirdPartyApp) {
+			thirdPartyApp.classList.add('rgh-bypass-link');
+			await bypass(thirdPartyApp);
```

Comment:
No need, `observe` isn't aware of it anyway
```suggestion
		add(thirdPartyApp) {
			thirdPartyApp.classList.add('rgh-bypass-link');
			void bypass(thirdPartyApp);
```